docs/development.rst

```diff
@@ -1,6 +1,91 @@
 .. _development:
 
+1. System requirement :
```

Comment:
should that be  `1. System requirement:` or is that blank before the colon needed, or do we need the colon at all?

Reply:
I will remove the colon. 👍🏻